core/iwasm/libraries/simde/simde.cmake

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2024 Amazon Inc.  All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# simde is a header only library
+
+set (LIB_SIMDE_DIR ${CMAKE_CURRENT_LIST_DIR})
+
+if (WAMR_BUILD_TARGET MATCHES "AARCH64.*" OR "ARM.*")
+  add_definitions (-DWASM_ENABLE_SIMDE=1)
+endif()
+else ()
+  add_definitions (-DWASM_ENABLE_SIMDE=0)
```

Comment:
Suggest to add `#ifndef WASM_ENABLE_SIMDE #define WASM_ENABLE_SIMDE 0 #endif` in core/config.h, and remove this line.